generators/server/templates/_build.gradle

```diff
@@ -288,7 +288,7 @@ dependencies {
     <%_ if (databaseType === 'sql') { _%>
     compile "org.hibernate:hibernate-entitymanager"
     compile "org.hibernate:hibernate-envers"
-    compile "org.hibernate:hibernate-validator"
+    compile "org.hibernate.validator:hibernate-validator"
```

Comment:
use new validator lib to validate new constrains as javax.validation.constraints.Email and javax.validation.constraints.NotNull

Reply:
Oh I had to remove the validation for blobs as it was failing - I need to test if I can revert my changes! Great!